views/scss/test-runner.scss

```diff
@@ -50,14 +50,36 @@
         &.horizontal-action-bar {
             opacity: 0;
 
-            .progress-box, .timer-box, .title-box {
-                padding-top: 4px
+            .title-box {
+                padding-top: 4px;
+            }
+            .progress-box, .timer-box, .item-number-box {
+                padding-top: 4px;
+                display: inline-block;
+                white-space: nowrap;
+                -webkit-flex: 0 0 auto;
+                flex: 0 1 auto;
+                .qti-controls {
+                    display: inline-block;
+                    margin-left: 20px;
+                    white-space: nowrap;
+                }
             }
             .progressbar {
                 margin-top: 5px;
                 min-width: 150px;
+                max-width: 200px;
                 height: 0.6em;
             }
+
+            &.top-action-bar.has-timers > .control-box {
```

Comment:
You might also need to remove the `.has-timers` because the progress bar isn't shown without a timer.
I suggest you to define also `flex-flow: row wrap` to ensure everything stays in the same line.